### DISCUSSION TITLE
blog/autohotkey-scripts-windows

### DISCUSSION THREAD

**Discussion by giscus** [GorvGoyl/Personal-Site-Gourav.io#29] [Announcements]
Discussion related to the article:

**Title:** Useful AutoHotkey scripts (Windows) for quick lookup, in-line calculator, remap keys, battery alert, and more
**URL:** https://gourav.io/blog/autohotkey-scripts-windows

**Comment by drazda1** [comment]
Thank you Gourav for sharing this, this is amazing!
I'm newbie to AHK and wanted to know if all of these can be run from a single AHK file? If so, can you please share how?
